### PR TITLE
Dimensions: Modify reliableTrDimensions support test to account for FF

### PR DESCRIPTION
**Pull request by timmywil**
Fixes gh-4529

### Summary ###

Firefox incorrectly (or perhaps [correctly](https://github.com/w3c/csswg-drafts/issues/4444)) includes table borders in computed
dimensions, but they are the only one. Workaround this by testing for it and
falling back to offset properties

### Checklist ###

* [x] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
* [x] New tests have been added to show the fix or feature works
* [x] Grunt build and unit tests pass locally with these changes
* [x] If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com

+47 bytes currently

### CODE REVIEW

File: src/css/support.js [hunk 1]
```diff
@@ -102,14 +102,23 @@ define( [
 		// set in CSS while `offset*` properties report correct values.
 		// Behavior in IE 9 is more subtle than in newer versions & it passes
 		// some versions of this test; make sure not to make it pass there!
+		//
+		// Support: Firefox 70+
+		// Only Firefox includes border widths
+		// in computed dimensions. (gh-4529)
```
Comment (mgol) [inline src/css/support.js:108]: That means we need to add a version of that test on `master` as well, don't we? Perhaps smaller as we don't care for Edge there.

Comment (timmywil) [reply src/css/support.js:108]: Good point. I'll make a PR for master as well.

**Review by mgol — APPROVED** [review]
Looks good, thanks!

**Review by mgol — APPROVED** [review]
LGTM
